src/Auth/Authenticator.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Auth;
+
+use Tempest\Auth\Exceptions\MissingIdentifiableException;
+
+interface Authenticator
+{
+    public function login(Identifiable $identifiable): void;
+
+    public function logout(): void;
+
+    /**
+     * @throws MissingIdentifiableException
+     */
+    public function user(): Identifiable|null;
```

Comment:
Shouldn't this method be called `identifiable`?

Reply:
Edit: maybe `getIdentifiable`. We'll refactor to use property hooks in 8.4, and it'll be easier to do search on `get*` methods in interfaces.